CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -97,9 +130,44 @@ function PlayerHospital:dailyAdviceChecks()
     self:giveAdvice(toilet_advice)
   end
 
-  -- Make players more aware of the need for radiators
-  if self:countRadiators() == 0 then
-    self:giveAdvice({_A.information.initial_general_advice.place_radiators})
+  if day == 10 then
+    self:warnForLongQueues()
+  end
+
+  -- Check for enough (well-placed) benches.
+  if day == 12 then
+    -- Compute average sitting ratio.
+    local sum_ratios = 0
+    local index = 1
+    while index <= num_sitting_ratios do
+      local ratio = self.adviser_data.sitting_ratios[index]
+      if ratio == nil then
+        sum_ratios = nil
+        break
+      else
+        sum_ratios = sum_ratios + ratio
+      end
+
+      index = index + 1
+    end
+
+    if sum_ratios ~= nil then -- Sufficient data available.
+      local ratio = sum_ratios / num_sitting_ratios
+      if ratio < 0.7 then -- At least 30% standing.
+        local bench_advice = {
+          _A.warnings.more_benches, _A.warnings.people_have_to_stand,
+        }
+        self:giveAdvice(bench_advice)
+
+      elseif ratio > 0.9 then
+        -- Praise having enough well placed seats about once a year.
+        local bench_advice = {
+          _A.praise.many_benches, _A.praise.plenty_of_benches,
+          _A.praise.few_have_to_stand,
+        }
+        self:giveAdvice(bench_advice, 1/12)
```

Comment:
```suggestion
        self:giveAdvice(bench_advice, 1 / 12)
```

Reply:
I think there might be a few of these with spaces needed that happy to fix anyway

Reply:
Finally done this one, sorry.